Tests/Shared/DIGraphTest.swift

```diff
@@ -18,13 +18,19 @@ import XCTest
  }
  ```
  */
-open class BaseDIGraphTest: IntegrationTest {
+open class BaseDIGraphTest: UnitTest {
```

Comment:
Same as above.

Updated the test to inherit from `UnitTest` instead because `Shared` module also has no longer access to any `IntegrationTest` and we can neither really initialize the SDK or any module in it. However, the test should work fine using `UnitTest` too.